packages/web-app-admin-settings/src/composables/actions/users/useUserActionsEdit.ts

```diff
@@ -0,0 +1,27 @@
+import { eventBus } from 'web-pkg'
+import { SideBarEventTopics } from 'web-pkg/src/composables/sideBar'
+import { useGettext } from 'vue3-gettext'
+import { computed } from 'vue'
+import { Action } from 'web-pkg/src/composables/actions'
+
+export const useUserActionsEdit = () => {
+  const { $gettext } = useGettext()
+
+  const actions = computed((): Action[] => [
```

Comment:
Why not `UserAction`? Then you don't need to cast `resources as unknown[]` down below.

```suggestion
  const actions = computed((): UserAction[] => [
```